test/service/test_repository_projects.py

```diff
@@ -65,6 +69,7 @@ def assert_file_type_summaries(hit):
             'fileTypeSummaries'
         }
         projects_properties = {
+            'accessible',
```

Comment:
I'm surprised that this is the only mention of the new property in the test code. Ideally, there should be a test case that asserts a project response with this set to `true` for one project and `false` for another.